src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Startup.cs

```diff
@@ -2,24 +2,127 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+#if (OrganizationalAuth || IndividualB2CAuth)
+using Microsoft.AspNetCore.Authentication;
+#endif
+#if (OrganizationalAuth)
+using Microsoft.AspNetCore.Authentication.AzureAD.UI;
+#if (MultiOrgAuth)
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+#endif
+using Microsoft.AspNetCore.Authorization;
+#endif
+#if (IndividualB2CAuth)
+using Microsoft.AspNetCore.Authentication.AzureADB2C.UI;
+#endif
 using Microsoft.AspNetCore.Builder;
+#if (IndividualLocalAuth)
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI;
+#endif
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
 #if (RequiresHttps)
 using Microsoft.AspNetCore.HttpsPolicy;
 #endif
+#if (OrganizationalAuth)
+using Microsoft.AspNetCore.Mvc.Authorization;
+#endif
+#if (IndividualLocalAuth)
+using Microsoft.EntityFrameworkCore;
+#endif
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+#if(MultiOrgAuth)
+using Microsoft.IdentityModel.Tokens;
+#endif
 using RazorComponentsWeb_CSharp.Data;
 
 namespace RazorComponentsWeb_CSharp
 {
     public class Startup
     {
+#if (OrganizationalAuth || IndividualAuth)
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+#endif
```

Comment:
You can avoid the `#if` block here and just add the ctor unconditionally, as that is what other templates do and we want to be aligned.

Reply:
Seems reasonable. Done.